proposals/0095-xof-hashing.md

```diff
@@ -0,0 +1,325 @@
+---
+simd: '0095'
+title: extendable output (XOF) hashing support
+authors:
+  - Ralph Ankele
+category: Standard
+type: Core
+status: Draft
+created: 2023-12-14
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This proposal introduces three new concepts to the Solana runtime:
+
+- Support extendable Output Functions (XOF) hashing, based on cSHAKE 
```

Comment:
BLAKE3 has XOF, could we not make the existing syscall for BLAKE3 just support XOF?